api-report/sequence.api.md

```diff
@@ -103,7 +103,7 @@ export interface IInterval {
     compareEnd(b: IInterval): number;
     compareStart(b: IInterval): number;
     // @internal
-    modify(label: string, start: number | undefined, end: number | undefined, op?: ISequencedDocumentMessage, localSeq?: number): IInterval | undefined;
+    modify(label: string, start: number | "start" | undefined, end: number | "end" | undefined, op?: ISequencedDocumentMessage, localSeq?: number): IInterval | undefined;
```

Comment:
don't some sequences of operations cause intervals with both endpoints on the start or both endpoints on the end? Seems like we need to support `number | "start" | "end" | undefined` here.